packages/models-library/src/models_library/services.py

```diff
@@ -115,7 +114,7 @@ class Config:
         frozen = True
 
 
-RunID = UUID
+RunID: TypeAlias = str
```

Comment:
I thought this was going to be an int?
I have just read the note in the PR. Perhaps you can add that note here ?